src/main/java/seedu/address/logic/commands/ViewCommentCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_STUDENTID;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.attendance.Comment;
+import seedu.address.model.person.NusNetId;
+import seedu.address.model.person.Student;
+import seedu.address.model.tutorial.Tutorial;
+import seedu.address.model.tutorial.TutorialName;
+
+public class ViewCommentCommand extends Command {
+    public static final String COMMAND_WORD = "view_comment";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views comments for a student. "
+            + "Parameters: "
+            + PREFIX_STUDENTID + "STUDENT ID "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_STUDENTID + "e01234567 ";
+
+    public static final String MESSAGE_SUCCESS = "Comment for student %s: %s";
+    public static final String MESSAGE_STUDENT_NOT_FOUND = "The specified student does not exist.";
+
+    private final NusNetId studentToViewComment;
+
+    /**
+     * Creates an CommentCommand to add a comment for the specified {@code Student}
+     */
+    public ViewCommentCommand(NusNetId studentId) {
+        studentToViewComment = studentId;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (!model.hasStudentWithId(studentToViewComment)) {
+            throw new CommandException(String.format(MESSAGE_STUDENT_NOT_FOUND));
+        }
+
+        Student student = model.getStudentWithId(studentToViewComment);
+        TutorialName tutorialName = student.getTutorialName();
+        Tutorial tutorial = model.getTutorialWithName(tutorialName);
+        Comment comment = model.getComment(tutorial, studentToViewComment);
+        return CommandResult.createStudentCommandResult(String.format(MESSAGE_SUCCESS, studentToViewComment, comment));
```

Comment:
If the CommandResult carries the displayable type of "STUDENT", we may not be able to see the comments in the displayed student list as comment is part of attendance. A few alternatives we can consider for the implementation of this feature:
1. Move the "Comment" attribute to "Student" so the when "view_comment" is called, we see a list of PersonCards belonging to the students with their associated comments. (As the PersonCard has too many attributes currently, I will create a StudentCard with specialised attributes so when we call "list_student", we display the list of students with just the comments, contact number and student id)
2. Remove ViewCommentCommand as current ListAttendanceCommand by class and week provides the weekly attendance along with the "Comment", so we will need to explain clearly on to access the comments in the User Guide.